Crashlytics/Crashlytics/Controllers/FIRCLSExistingReportManager.m

```diff
@@ -17,23 +17,29 @@
 #import "Crashlytics/Crashlytics/Controllers/FIRCLSManagerData.h"
 #import "Crashlytics/Crashlytics/Controllers/FIRCLSReportUploader.h"
 #import "Crashlytics/Crashlytics/DataCollection/FIRCLSDataCollectionToken.h"
+#import "Crashlytics/Crashlytics/Helpers/FIRCLSLogger.h"
 #import "Crashlytics/Crashlytics/Models/FIRCLSFileManager.h"
 #import "Crashlytics/Crashlytics/Models/FIRCLSInternalReport.h"
 #import "Crashlytics/Crashlytics/Private/FIRCrashlyticsReport_Private.h"
 #import "Crashlytics/Crashlytics/Public/FirebaseCrashlytics/FIRCrashlyticsReport.h"
 
+// This value should stay in sync with the Android SDK
+const NSUInteger MAX_UNSENT_REPORTS = 4;
```

Comment:
Should this be extern'ed? It's mentioned in the header, but doesn't seem to be defined there.